admin/server/github.go

```diff
@@ -836,11 +836,18 @@ func (s *Server) fetchReposForInstallation(ctx context.Context, client *github.C
 			if repo.Owner != nil {
 				owner = fromStringPtr(repo.Owner.Login)
 			}
+			var branch string
+			if repo.DefaultBranch != nil {
```

Comment:
Just confirming, did you check if either one of them is always set ? The github APIs though document same repository schema but I have seen usually they miss field in one API or other.

Reply:
Good question, just checked [the docs](https://docs.github.com/en/rest/apps/installations?apiVersion=2022-11-28#list-repositories-accessible-to-the-user-access-token) for this API. `default_branch` is in required section.